README.md

```diff
@@ -1,8 +1,40 @@
 # example-finder
 
+## Build:
+Dependencies:
+```
+gcc (required for sqlite3 support)
+
+Fedora:
+dnf install -y gcc
+
+Windows:
+http://mingw-w64.org/doku.php/download#mingw-builds
+```
+build:
+
+```
+go build
+```
+
+test:
+```
+go test ./...
+```
+
 *Work in progress*
 ## Usage:
 ```
-  ./example-finder search text [-l|--lang LANG] [-t|--token TOKEN]
+./example-finder search TEXT [-l|--lang LANG] [-t|--token TOKEN] [-m|--mode MODE] [-r|--results num]
+  lang    - language to return results in (not implemented yet)
+  token   - Github API token, see https://github.com/settings/tokens - should be inside .token file
+  mode    - currently only REST mode is supported, a future plan is to support GraphQL as well
+  results - number of results per page to return (default 30)
+
+Example usage:
+./example-finder search go-pg -r 100 -t ABC
```

Comment:
explain -t/token with example. you're saying it's a token, yet talking about file in description. is ABC the entire token? is it the name of the file? If there is no way to pass a file, specify it's taken from .token file if not given (assuming it is correct behavior)